scripts/event.js

```diff
@@ -47,44 +53,54 @@
       // No user is signed in.
     }
   });
+  // TODO: Sign out button
 
+  // writes to firebase
   Event.writetoFB = function (event) {
     console.log('saving to firebase');
     var newEvent = firebasedb.ref('events').push()
     newEvent.set(event)
   };
 
+  // looks up the lat and lng of a zip code
   Event.lookupZip = function (zip) {
     return firebasedb.ref('/publicInfo/zips/' + zip).once('value').then(function(snapshot) {
-      var location = new google.maps.LatLng(snapshot.val().LAT,  snapshot.val().LNG)
+      var location = new google.maps.LatLng(snapshot.val().LAT, snapshot.val().LNG)
       Event.returnNearest(location)
     })
   }
 
+  // given a zip code return the nearest event
   Event.returnNearest = function (location) {
     firebase.database().ref('/events/').once('value').then(function(snapshot) {
```

Comment:
I've alway used `.ref` without a leading and trailing `/` on the path. Not sure if it makes a difference.